platforms/common/include/px4_platform_common/px4_work_queue/WorkQueueManager.hpp

```diff
@@ -73,7 +73,7 @@ static constexpr wq_config_t INS1{"wq:INS1", 6000, -15};
 static constexpr wq_config_t INS2{"wq:INS2", 6000, -16};
 static constexpr wq_config_t INS3{"wq:INS3", 6000, -17};
 
-static constexpr wq_config_t hp_default{"wq:hp_default", 1900, -18};
+static constexpr wq_config_t uavcan{"wq:uavcan", 4000, -16};
```

Comment:
@kthompson11 this looks like a merge error

Reply:
Agreed. The wrong line got changed in the merge.